libs/comparison/src/lib/comparison-app.component.less

```diff
@@ -98,3 +98,14 @@
   align-items: flex-end;
   justify-content: flex-end;
 }
+
+/*
+******************************************
+Right side panel
+******************************************
+*/
+.gd-difference {
```

Comment:
why is it here? not in the difference.component.less

Reply:
Nice catch! I already moved this part to the difference.component.less and removed it here, but didn't saved the file before push.